UT4MasterServer/Other/EpicID.cs

```diff
@@ -27,14 +27,14 @@ private EpicID(string id)
 	{
 		if (id.Length != 32)
 		{
-			throw new InvalidEpicIDException("ID needs to be 32 characters long.");
+			throw new InvalidEpicIDException("ID needs to be 32 characters long.", id, 18009, "errors.com.epicgames.account.id_not_32_characters");
```

Comment:
are the numerical error codes taken from somewhere or did you come up with them?
i think it should be the same as original when `errors.com.epicgames.modules.stats.invalid_account_id` or similar error is triggered on official servers